python-lib/dku_utils/node_pool.py

```diff
@@ -28,4 +30,49 @@ def get_node_pool_args(node_pool):
         args = args + ["--ssh-access"]
         args = args + ['--ssh-public-key', node_pool.get('publicKeyName', '')]
         
-    return args
\ No newline at end of file
+    return args
+
+def get_node_pool_yaml(node_pool):
+    yaml = {}
+    if 'machineType' in node_pool:
+        yaml['instanceType'] = node_pool['machineType']
+    yaml['volumeType'] = node_pool.get('diskType', 'gp2')
```

Comment:
Here is the line where I change the default to `gp2`. I can revert that to `gp3` amd instead change the form.

Reply:
I think that makes sense. It is possible that the EKS default changes eventually, but that's not a big deal for now. I'll update the form to reflect that we will use the default imposed by EKS instead of mentioning gp2 instead